bigquery_etl/stripe/allowed_fields.py

```diff
@@ -33,11 +33,24 @@ def _valid_float(value: str):
 
 
 def get_rooted_schema(type_) -> bigquery.SchemaField:
-    """Load schema for given stripe type from json."""
+    """Load schema for given stripe type from json.
+
+    If the schema file does not exist, return the schema for data.{type_} from the event
+    schema.
+    """
     path = Path(__file__).parent / f"{type_}.schema.json"
-    return bigquery.SchemaField.from_api_repr(
+    schema_exists = path.exists()
+    if not schema_exists:
+        path = Path(__file__).parent / "event.schema.json"
+    root = bigquery.SchemaField.from_api_repr(
         {"name": "root", "type": "RECORD", "fields": ujson.loads(path.read_text())}
```

Comment:
Does the `name` field at the root matter?  Because this won't be exactly equivalent to what was done before, where the root always had `"name": "root"`.  That'll now only be the case for events, and all other entities will have `"name": "<entity name>"`.

Reply:
it doesn't get used